docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
+https://www.ansible.com/overview/how-ansible-works[Ansible^] is an open source configuration management and automation tool.
+
+Ansible uses _modules_ to execute tasks. The output of one module can be the input for the next one or they can be independent of each other.
+
+You can combine one or more Ansible modules to make a play. Combine two or more plays to create an Ansible playbook, a list of sequential tasks (written in YAML) that automatically executes when you run the `ansible-playbook` command. You can use an Ansible playbook to describe the desired state of your infrastructure and then have Ansible provision it.
+
+An Ansible role is a special kind of playbook that is fully self-contained and portable with the tasks, variables, configuration templates, and other supporting files that are needed to complete your chosen tasks. Multiple roles can exist inside a collection, allowing easy sharing of content through Automation Hub and https://galaxy.ansible.com[Ansible Galaxy^].
+
+//Additional line break to resolve mod docs generation error
+
+[id="proc-install-rhoas-ansible-collection_{context}"]
+=== Installing the RHOAS Ansible collection
+
+[role="_abstract"]
+You can use the https://galaxy.ansible.com/rhoas/rhoas[RHOAS Ansible collection] to fully manage your Kafka environment with Ansible. You can create, view, configure, and delete the following {product-long-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+.Prerequisites
+
+* https://www.python.org/downloads/[Python 3.9^] or later is installed.
+* https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Ansible 2.9^] or later is installed.
+* Some additional dependencies required by this collection are installed. To install these, enter the following commands:
++
+[source,shell]
+----
+$ pip install rhoas-sdks
+$ pip install python-dotenv
+----
+NOTE: Use the `pip install rhoas-sdks --force reinstall` command to ensure that the most up to date versions of the SDKs are installed.
++
+
+* This collection works best when used in a Python virtual environment. To create and activate a Python virtual environment, enter the following command:
++
+[source,shell]
+----
+$ python3 -m venv rhoas/
+. rhoas/bin/activate
+----
++
+
+.Procedure
+
+. Open a new terminal window.
+. Install the {product-long-rhoas} (RHOAS) Ansible collection.
++
+[source,shell]
+----
+$ ansible-galaxy collection install rhoas.rhoas
+----
++
+By default, the `ansible-galaxy collection install` command uses \https://galaxy.ansible.com as the Galaxy server URL. The URL is specified as the value of the `GALAXY_SERVER` parameter in the `ansible.cfg` configuration file.
++
+A message displays that the RHOAS collection was installed successfully to the following path: `home/<usr>/.ansible/collections/ansible_collections/rhoas/rhoas`. All modules for the collection can be found in this location.
```

Comment:
'Display' is a transitive verb that requires an object. Suggest replacing with 'states':

"A message states...."

Reply:
Italicize `<usr>`.